src/dbus/server/dbus_agent.cpp

```diff
@@ -71,7 +71,7 @@ void DBusAgent::Init(void)
     {
     case OT_COPROCESSOR_RCP:
         mThreadObject = MakeUnique<DBusThreadObjectRcp>(*mConnection, mInterfaceName,
```

Comment:
if RCP object, will it cause feature divergency between RCP and NCP. feature parity between these two modes should be expected? @Irving-cl

Reply:
NCP currently only has very limited feature and it's not ready to support ePSKc in it now. Let's first implement it in RCP first. I'll work with @mia1yang on this later when it's ready.